readme1.md

```diff
@@ -0,0 +1,76 @@
+..........here are possible techniques for generating test cases from api schema...............
+
+Swagger/OpenAPI: Swagger or OpenAPI is a specification for describing RESTful APIs. We can use a tool like Swagger Codegen to generate client and server code for the API, including tests.
+Pros:
+
+Standardized specification.
+Widely used and supported.
+Easy to use.
+Cons:
+
+Limited to RESTful APIs.
+May not cover all test cases.
+Fuzzing: Fuzzing involves sending random or semi-random inputs to the API to see how it responds. This can help to uncover unexpected or edge cases that may not be covered by more traditional testing approaches.
+Pros:
+
+Can uncover unexpected or edge cases.
+Can be automated.
+Cons:
+
+Can be time-consuming and resource-intensive.
+May not cover all test cases.
+Contract Testing: Contract testing involves creating a contract or agreement between the client and the server that specifies the expected behavior of the API. We can use a tool like Pact to generate automated tests based on the contract.
+Pros:
+
+Tests are based on a shared understanding of the API.
+Can be automated.
+Cons:
+
+Requires agreement and cooperation between client and server teams.
+May not cover all test cases.
+Swarm Testing: Swarm testing involves generating a large number of tests that cover a wide range of inputs and conditions. This can be done using a tool like Dredd, which can generate tests based on the API schema.
+Pros:
+
+Covers a wide range of inputs and conditions.
+Can be automated.
+Cons:
+
+Can be time-consuming and resource-intensive.
+May not cover all test cases.
+
```

Comment:
Good work with exploring different techniques for generating test cases from api schema . But I guess there is no that good technique in the solutions that you mentioned above . Because every technique has a con that it may not cover all cases or edge cases . Majorly that is the problem we are trying to solve over here with this task. As It is a tiresome process for user to write edge cases we are introducing this so that user will not have to do much.